Hush/Assets/Scripts/Relics/RelicDome.cs

```diff
@@ -47,17 +43,27 @@ private void OnTriggerEnter(Collider other)
             if (!other.gameObject.CompareTag(Tags.Player)) 
                 return;
 
+            // show interaction text
+            if (other.gameObject.CompareTag(Tags.Player))
+            {
+                transform.GetChild(0).gameObject.SetActive(true);
```

Comment:
same as other document, i think a reference to the object would be  better. applies to everything